src/test/resources/net/sf/jabref/logic/importer/fileformat/RisImporterTestDoiAndJournalTitle.bib

```diff
@@ -0,0 +1,10 @@
+@patent{,
+  author   = {some author},
+  month    = dec,
+  year     = {2014},
+  caption  = {254},
+  edition  = {5},
+  editor   = {some editor},
+  language = {eng},
+  number   = {25},
+}
```

Comment:
The name of the test files confuses me a lot...Isn't the purpose here to test the journal title and doi fields? So why are the fields not present in the file?